README.md

```diff
@@ -1,24 +1,31 @@
-[![Join the chat at https://gitter.im/zlsa/atc](https://badges.gitter.im/Join%20Chat.svg)](https://gitter.im/zlsa/atc?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge&utm_content=badge)
+<p align="center">
+    <a href="https://gitter.im/zlsa/atc?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge&utm_content=badge"><img src="https://badges.gitter.im/Join%20Chat.svg"/></a>
+    <a href="https://github.com/openscope/openscope/tree/master"><img src="https://img.shields.io/travis/openscope/openscope/master.svg"/></a>
+    <a href="https://github.com/openscope/openscope/releases"><img src="https://img.shields.io/github/release/openscope/openscope.svg"/></a>
+    <img src="https://img.shields.io/badge/license-MIT-blue.svg"/>
+</p>
+
 ## Air Traffic Control
 
 An ATC simulator in HTML5.
 
 ## Running Locally
-It is possible to run this application locally.  You will need to have node and npm install in order for this to work.  For more information, view the [Tools Documentation](tools/README.md)
+It is possible to run this application locally. You will need to have `node` and `npm` installed in order for this to work. For more information, view the [Tools Documentation](tools/README.md).
 
 ## Privacy
 
-Page hits are tracked with Google Analytics; feel free to use an
+Page hits are tracked with Google Analytics, so feel free to use an
 adblocker. The tracking is used only to get the number of page hits.
 
 ## Contributing
 
-Just create a pull request and I'll get to it. Thanks for helping!
+Just create a pull request and we'll get to it. Thanks for helping!
```

Comment:
@erikquinn what are your thoughts on this section?

I'd like to get rid of forks entirely, and just have people ask to contribute.  But I know its much easier to dip your feet in the pool when you can do so via forks.